tortoise/verifying_tortoise_test.go

```diff
@@ -324,21 +340,42 @@ func createTurtleLayer(l types.LayerID, msh *mesh.DB, bbp baseBlockProvider, ivp
 	return lyr
 }
 
+//func size(m map[types.LayerID]map[types.BlockID]Opinion) int {
```

Comment:
Remove comment

Reply:
It will be cool to have in tortoise error messages layer_id also.